recipes/boost/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "1.60.0" %}
+{% set filename = "boost_%s.tar.bz2" % version.replace(".", "_") %}
+
+package:
+  name: boost
+  version: {{ version }}
+
+source:
+  fn:  {{ filename }}
+  url: http://sourceforge.net/projects/boost/files/boost/{{ version }}/{{ filename }}
```

Comment:
Cool, glad we like this. If you guys have more thoughts on how to make things more readable still, please share.